app/src/androidTest/java/org/mozilla/tv/firefox/helpers/MainActivityTestRule.kt

```diff
@@ -35,7 +34,6 @@ class MainActivityTestRule(
      * implementation changes and the tests break. In that case, this code might be
      * broken because it's not used, and thus tested, at present.
      */
-    @VisibleForTesting
     lateinit var loadingIdlingResource: SessionLoadedIdlingResource
```

Comment:
Nit: I get why we need to relax the visibility on this, but I think we should put a private set on it.